mlflow/openai/utils.py

```diff
@@ -65,8 +66,9 @@ def request(*args, **kwargs):
         else:
             url = kwargs.get("url")
 
-        if "chat/completions" in url:
-            return _mock_chat_completion_response()
+        if re.match(r"^https://api\.openai\.com/v\d+/chat/completions$", url):
+            messages = json.loads(kwargs.get("data")).get("messages")
+            return _mock_chat_completion_response(content=json.dumps(messages))
```

Comment:
To check a request is sent with an expected `messages`, return a dumped `messages` as a response.